presto-cli/src/main/java/io/prestosql/cli/Console.java

```diff
@@ -140,7 +140,7 @@ public boolean run()
                 Optional.ofNullable(clientOptions.krb5CredentialCachePath),
                 !clientOptions.krb5DisableRemoteServiceHostnameCanonicalization)) {
             if (hasQuery) {
-                return executeCommand(queryRunner, query, clientOptions.outputFormat, clientOptions.ignoreErrors);
+                return executeCommand(queryRunner, query, clientOptions.outputFormat, clientOptions.ignoreErrors, clientOptions.debug);
```

Comment:
Rather than using `--debug`, which has existing behavior like showing stack traces, I think we should add a new `--progress` option to show progress in batch mode.